lib/ui/addon/components/sidebar-toggle.js

```diff
@@ -1,21 +1,14 @@
-import { equal } from '@ember/object/computed';
 import Component from '@ember/component';
-export default Component.extend({
+import layout from '../templates/components/sidebar-toggle';
 
+export default Component.extend({
+  layout,
   tagName: 'button',
-
-  side: 'right',
-
-  isExpanded: false,
-
-  isRight: equal('side', 'right'),
```

Comment:
No need for this right stuff anymore?

Reply:
As far as I can tell we never had a sidebar toggle for the left sidebar. We can add this back if we ever need it.